acceptance-tests/tests/src/test/resources/jsonrpc/engine/paris/test-cases/02_get_payload.json

```diff
@@ -18,12 +18,12 @@
       "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
       "prevRandao": "0x0000000000000000000000000000000000000000000000000000000000000000",
       "blockNumber": "0x1",
-      "gasLimit": "0x1c9c380",
+      "gasLimit": "0x1ca35ef",
```

Comment:
the gasLimit has changed because the new limit is 36M: We start with a block of 30M and increase gas limit for the next block by 30M / 1024.